include/aws/nitro_enclaves/kms.h

```diff
@@ -971,14 +971,18 @@ void aws_nitro_enclaves_kms_client_destroy(struct aws_nitro_enclaves_kms_client
  * This function generates an Attestation Document and calls AWS KMS with enclave-specific parameters.
  * Calling it from a non-enclave environment will fail.
  *
- * @param[in]   client      The AWS KMS client to use for calling the API.
- * @param[in]   ciphertext  The ciphertext to decrypt.
- * @param[out]  plaintext   The plaintext output of the call. Should be an empty, but non-null aws_byte_buf.
- * @return                  Returns AWS_OP_SUCCESS if the call succeeds and plaintext is populated.
+ * @param[in]   client                  The AWS KMS client to use for calling the API.
+ * @param[in]   key_id                  The ARN or alias of AWS KMS CMK used to encrypt the data key.
+ * @param[in]   encryption_algorithm    The encryption algorithm that will be used to decrypt the ciphertext.
+ * @param[in]   ciphertext              The ciphertext to decrypt.
```

Comment:
Please add some notes that `ciphertext` may already contain key id as a metadata for symmetric keys and in this case `key_id` and `encryption_algorithm` may be `NULL`